### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by joe83830**
jun-hsiang-liao jl3744

### CODE REVIEW

**Review by amfleming — CHANGES_REQUESTED** [review]
You need to pull from upstream to avoid conflicts

And don't delete the Alpha-order comment

**Comment by joe83830** [comment]
I modified the files

File: README.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ People
 
 
 #### Students
+
```
Comment (amfleming) [inline README.md]: Why new line?

File: README.md [hunk 2]
```diff
@@ -58,6 +59,7 @@ People
 * [Rom Cohen](people/rom-cohen.md) - rc783 - [https://cs5356-rc783.s3-website.us-east-2.amazonaws.com/grading.html](https://cs5356-rc783.s3-website.us-east-2.amazonaws.com/grading.html)
 * [Marika Cusick](people/marika-cusick.md) - mmc265 - [https://s3.us-east-2.amazonaws.com/marikacusickbucket/graded.html](https://s3.us-east-2.amazonaws.com/marikacusickbucket/graded.html)
 * [Jonathan Cutler](people/jonathan-cutler.md) - jlc553 - [https://s3.us-east-2.amazonaws.com/cs5356-jlc553/grading.html](https://s3.us-east-2.amazonaws.com/cs5356-jlc553/grading.html)
+* [jun-hsiang-liao](people/jun-hsiang-liao.md) - jl3744 - [http://jl3744.xyz.s3-website.us-east-2.am    azonaws.com](http://jl3744.xyz.s3-website.us-east-2.amazonaws.com/grading.html)
```
Comment (amfleming) [inline README.md]: Shouldn't this start with `[Jun Liao]` ?

File: README.md [hunk 3]
```diff
@@ -97,6 +99,7 @@ People
 * [Elena Zhizhimontova](people/elena-zhizhimontova.md) - ez256 - [https://s3-us-west-2.amazonaws.com/zhizhimontova.com/grading.html](https://s3-us-west-2.amazonaws.com/zhizhimontova.com/grading.html)
 * [Yuxiang Zhu](people/yuxiang-zhu.md) - yz2329 - [https://s3.amazonaws.com/cs5356-yz2329/grading.html](https://s3.amazonaws.com/cs5356-yz2329/grading.html)
 
+>>>>>>> 67600b5a447043068f7b780c576b13d67e0341f4
```
Comment (amfleming) [inline README.md]: What's this?

File: people/jun-hsiang-liao.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+Adam Fleming
```
Comment (amfleming) [inline people/jun-hsiang-liao.md]: Shouldn't this say `Jun Liao`?

**Review by amfleming — CHANGES_REQUESTED** [review]
Yikes

**Comment by joe83830** [comment]
Sry I removed the weird thing on line 102.
I also Capitalized my name and removed the dashes in between.
But actually my name is Jun-Hsiang Liao, it's Mandarin so there are two words in my first name and this is how it appears on my passport.
I changed it to Jun Hsiang Liao.
Or should I combine it into sth like JunHsiang Liao?
Sorry for the trouble :P

**Comment by amfleming** [comment]
Hi! You can spell your name however you want

I'm certain you don't spell it `Adam Fleming` however

**Comment by joe83830** [comment]
I think I corrected the format.
This is my first time forking things on GitHub, hope I finally got everything working, thanks!

File: people/junhsiang-liao.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Adam Fleming
```
Comment (amfleming) [inline people/junhsiang-liao.md]: Still says `Adam Fleming` 😬

**Review by amfleming — CHANGES_REQUESTED** [review]
Hi! Still have an accidental new line 

And people/junhsiang-liao.md still stays 'Adam Fleming'

**Comment by joe83830** [comment]
Noooooo I'm terribly sorry sir I finally understood what you meant!

**Pull request merged**